new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMapOverride.java

```diff
@@ -0,0 +1,679 @@
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.domainExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.isHelmReleaseDeployed;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.operatorIsRunning;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.podExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.podReady;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.serviceExists;
+import static oracle.weblogic.kubernetes.utils.ExecCommand.exec;
+import static oracle.weblogic.kubernetes.utils.FileUtils.checkDirectory;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+// Test to create model in image domain and verify the domain started successfully
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Test to create model in image domain and start the domain")
+@IntegrationTest
+class ItMiiConfigMapOverride implements LoggedTest {
+
+  // mii constants
+  private static final String WDT_MODEL_FILE = "model1-wls.yaml";
+  private static final String MII_IMAGE_NAME = "mii-image";
+  private static final String APP_NAME = "sample-app";
+
+  // domain constants
+  private static final String DOMAIN_VERSION = "v7";
+  private static final String API_VERSION = "weblogic.oracle/" + DOMAIN_VERSION;
+
+  private static final String READ_STATE_COMMAND = "/weblogic-operator/scripts/readState.sh";
+
+  private static HelmParams opHelmParams = null;
+  private static V1ServiceAccount serviceAccount = null;
+  private String serviceAccountName = null;
+  private static String opNamespace = null;
+  private static String operatorImage = null;
+  private static String domainNamespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+  private static String dockerConfigJson = "";
+
+  private String domainUid = "domain1";
+  private String miiImage = null;
+  private StringBuffer checkJdbc = null;
+  private V1Patch patch = null;
+
+  /**
+   * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(5, MINUTES).await();
+
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
```

Comment:
you are not creating it, it already exists

Reply:
can be changed to "Assigning...."

Reply:
logger.info("Assigning unique namespace for Operator");
 logger.info("Assigning unique namespace for Domain");